sdk/scheduler/src/test/resources/invalid-rlimit-legacy-name.yml

```diff
@@ -0,0 +1,26 @@
+name: "data-store"
+pods:
+  meta-data:
+    count: 2
+    container:
+      rlimits:
+        RLIMIT_NONSENSE:
+          soft: 5
+          hard: 10
+    tasks:
```

Comment:
If you're trying to provide an example of a particular invalid scenario, please get rid of all the non-essential stuff here.  `cmd, port, volumes, envvars` etc.